MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -1270,21 +1291,31 @@ private void OnSetSoundState(SetSoundState payload, Action onCompleteCallback)
 
         public bool CheckIfSoundExpired(Guid id)
         {
-            if (_soundInstances != null && _soundInstances.TryGetValue(id, out AudioSource soundInstance))
+            if (_mediaInstances != null && _mediaInstances.TryGetValue(id, out System.Object mediaInstance))
```

Comment:
Combine into conditional access: `_mediaInstances?.TryGetValue`

Reply:
I think it's less readable that way, so I'm leaving it the way it is.

---
In reply to: [302638875](https://github.com/Microsoft/mixed-reality-extension-unity/pull/86#discussion_r302638875) [](ancestors = 302638875)